plugins/com.google.cloud.tools.eclipse.appengine.newproject.maven/helpContexts.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<contexts>
+  <context id="NewMavenProjectContext" title="Create Maven-based App Engine Standard Project">
+     <description>Create a new Maven-based Eclipse project that can be deployed to the App Engine standard environment.
+
+&lt;b&gt;Group ID:&lt;/b&gt; The Maven Group ID. Should be an alphanumeric path separated by periods.
+
+&lt;b&gt;Artifact ID:&lt;/b&gt; The Maven Artifact ID. Should be an alphanumeric name separated by dashes.
+ 
+&lt;b&gt;Libraries to add to build path:&lt;/b&gt; Add some libraries on the App Engine standard environment to your build path.</description>
```

Comment:
_Adds useful libraries for the App Engine standard environment for your project build path._